starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,47 @@
+body {
+  font-family: 'Open Sans';
+  font-size: 18px;
+  color: #444;
+  margin:3% 16% 3% 16%;
```

Comment:
This is an interesting way to center al the contents on the page.  It won't work on a larger project where some things are centered and some are not, but works fine here.